src/core/Table/Table.test.tsx

```diff
@@ -563,3 +562,42 @@ it('should show message when there is no data after filtering', () => {
   expect(rows.length).toBe(0);
   screen.getByText('No results. Clear filter.');
 });
+
+it('should not trigger sorting when filter is clicked', () => {
```

Comment:
If #70 gets merged, will this test be necessary?

Reply:
there is no test for this file, though it would be nice to have to cover more usecase

Reply:
Merged.